database/docker-compose.yml

```diff
@@ -6,6 +6,8 @@ services:
       POSTGRES_DB: waect
       POSTGRES_USER: user
       POSTGRES_PASSWORD: pass
+      DB_HOST: host.docker.internals
```

Comment:
Is this part of how you fixed the problem?

Reply:
yes